### ISSUE TITLE
Non-specific ground-glass opacification on pulmonary HRCT

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6572]
**Preferred term label:**  Non-specific ground-glass opacification on pulmonary HRCT

**Synonyms**

**Definition (free text, please give PubMed ID)**
Hazy area of increased attenuation in the lung with preserved visualisation of the blood vessels which does not indicate a specific diagnosis

**Parent term (use hpo.jax.org/app)**
Ground-glass opacification on pulmonary HRCT HP:0025179

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Pneumonitis
Pulmonary oedema
Adenocarcinoma

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  The phenotypes are not meant to be connected to specific diseases as HPO terms. It seems one could just use the parent term here? Or is there some phenotypic distinction.
